### PR TITLE
Add mentorship view (#16)

### LINKED ISSUE #16: Create Mentorship View based on Wireframe
Create the Mentorship View based on the wireframe. Put it inside of a WebView and make sure navigation is functional. Should be linked to mentorship.bitca.mp.

### PR DESCRIPTION
**Pull request by jsjqian**
Resolves #16 

Adds a WebView to contain the mentorship.bitca.mp page. 
Currently working for Android, will need someone to test it on iOS to ensure functionality.
Most of this code is pulled straight from https://facebook.github.io/react-native/docs/webview.html
Much of the styling needs to be redesigned, that is up to @AyeAndy 
Still need to eventually redesign the mentorship page

### CODE REVIEW

**Comment by goccert25** [comment]
LGTM on iOS

File: scenes/MentorsScene.js [hunk 1]
```diff
@@ -1,18 +1,186 @@
 import React, { Component } from 'react';
-import { View, Text } from 'react-native';
+import { View, Text, WebView, ScrollView, TextInput, TouchableWithoutFeedback, TouchableOpacity, StyleSheet } from 'react-native';
+
+var HEADER = '#1A2E33';
```
Comment (jdai8) [inline scenes/MentorsScene.js]: Can we put these into the shared styles file?

File: scenes/MentorsScene.js [hunk 2]
```diff
@@ -1,18 +1,186 @@
...
+var BGWASH = 'rgba(255,255,255,0.8)';
+var DISABLED_WASH = 'rgba(255,255,255,0.25)';
+
+var TEXT_INPUT_REF = 'urlInput';
```
Comment (jdai8) [inline scenes/MentorsScene.js]: we never use this

File: scenes/MentorsScene.js [hunk 3]
```diff
@@ -1,18 +1,186 @@
...
+          <TouchableOpacity
+            onPress={this.goForward}
+            style={this.state.forwardButtonEnabled ? styles.navButton : styles.disabledButton}>
+            <Text>
+              {'>'}
+            </Text>
+          </TouchableOpacity>
+        </View>
+        <WebView
+          ref={WEBVIEW_REF}
```
Comment (jdai8) [inline scenes/MentorsScene.js]: Use a callback function instead
https://facebook.github.io/react/docs/more-about-refs.html#the-ref-string-attribute

File: scenes/MentorsScene.js [hunk 4]
```diff
@@ -1,18 +1,186 @@
 import React, { Component } from 'react';
-import { View, Text } from 'react-native';
+import { View, Text, WebView, ScrollView, TextInput, TouchableWithoutFeedback, TouchableOpacity, StyleSheet } from 'react-native';
```
Comment (jdai8) [inline scenes/MentorsScene.js]: 80 character lines

**Review by jdai8 — CHANGES_REQUESTED** [review]

**Review by jdai8 — APPROVED** [review]

**Pull request merged**